lib/AST/ASTContext.cpp

```diff
@@ -5732,6 +5732,48 @@ ASTContext::getClangTypeForIRGen(Type ty) {
   return getClangTypeConverter().convert(ty).getTypePtrOrNull();
 }
 
+const clang::TypedefType *ASTContext::getTypedefForCXXCFOptionsDefinition(
```

Comment:
I'd prefer not to have Clang-specific APIs on `ASTContext`. Can we keep this within the ClangImporter, e.g., by making it a static method there?

Reply:
Sounds good, I made it a static method of `ClangImporter`